### DISCUSSION TITLE
Offline compiler Carbon

### DISCUSSION THREAD

**Discussion by Cobolorum** [carbon-language/carbon-lang#1523] [Getting started] [answered]
Is it possible to build a carbon compiler without all this online things such as: brew,  bazelisk, bazel ?

**Comment by Jark5455** [comment]
Maybe release precompiled versions of the carbon compiler?

**Reply by Cobolorum** [reply]
I agree with you that any offline compiler that even works poorly or is limited in functionality would give much more than any online emulator.

**Comment by jonmeow** [comment, marked as answer]
@Cobolorum I understand you mean well, but please don't use insulting words. Please review our [code of conduct](https://github.com/carbon-language/carbon-lang/blob/trunk/CODE_OF_CONDUCT.md).

http://carbon.compiler-explorer.com/ can be used, but keep in mind [Carbon is not ready for use](https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/faq.md#how-soon-can-we-use-carbon). The reason you're not finding a compiler is because none exists today (just an experimental interpreter).
